plugins/packages/packages_inspector.rb

```diff
@@ -90,12 +90,22 @@ def inspect_dpkg
       )
     end
     packages.each_slice(100) do |packages_slice|
-      apt_cache_output = @system.run_command(
-        "apt-cache",
-        "show",
-        *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
-        stdout: :capture
-      )
+      begin
+        apt_cache_output = @system.run_command(
+          "apt-cache",
+          "show",
+          *packages_slice.map { |p| "#{p.name}=#{[p.version, p.release].compact.join("-")}" },
+          stdout: :capture
+        )
+      rescue
```

Comment:
We could try to determine the version, but since it just calls the command again a little different and if it fails again, the inspection stops so I think it is Ok.